### DISCUSSION TITLE
Pdf vertical text conversion picture display is abnormal

### DISCUSSION THREAD

**Discussion by Forest-Wei** [dlemstra/Magick.NET#1373] [Help]
Original pdf:
![image](https://user-images.githubusercontent.com/31173376/231453305-a430d042-b17d-42ae-9764-09a6174c0e3c.png)

Converted image:
![image](https://user-images.githubusercontent.com/31173376/231452900-e06af581-225e-42a1-ac22-d5a744f581fe.png)

**Comment by dlemstra** [comment]
This seems to be a Ghostscript issue so you will need to ask them for help?

**Reply by Forest-Wei** [reply]
Thanks, man. I'll try.
